DemoController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DemoController : MonoBehaviour
+{
+    public GameObject TalkingHead;
+    public GameObject TalkBubble;
+    public GameObject CloseButton;
+
+    public Button[] TooltipButtons;
+
+    public string AnimationToPlay;
+
+    [Header("Debug & Unit Testing")]
+    public string MyDebugAnimation;

```

Comment:
Ah, this specific setup was put in place to simulate a CAPI call within the editor, which normally doesn't happen because of the way Unity phantoms out messages to outside sources.